client/jetpack-cloud/sections/partner-portal/license-product-card/style.scss

```diff
@@ -0,0 +1,72 @@
+.license-product-card {
+	width: 33%;
+}
+
+.license-product-card__inner {
+	margin: 0.5rem;
+	padding: 1.5rem;
+	border: 1px solid #ddd;
+	border-radius: 2px;
+	cursor: pointer;
+}
+
+.license-product-card.selected .license-product-card__inner {
+	border: 1px solid #fff;
+	background: #fff;
+	box-shadow: 0 0 12px #ddd;
+}
+
+.license-product-card__title {
+	font-size: 1.5rem;
+	margin-bottom: 1rem;
+	color: #444;
+	display: inline-block;
+}
+
+.license-product-card__radio {
+	border: 2px solid #444;
+	width: 16px;
+	height: 16px;
+	display: inline-block;
+	float: right;
+	border-radius: 10px; /* stylelint-disable-line */
+}
+
+.license-product-card__radio .gridicon {
+	color: #fff;
+	position: relative;
+	top: -4px;
+	left: 1px;
+	width: 14px;
+}
+
+.license-product-card.selected .license-product-card__radio {
+	border: 2px solid green;
+	background: green;
```

Comment:
The Figma file uses `#069e08` as the green colour, but the "Select license" button uses `var(--studio-jetpack-green-50);` (it looks like `#058415`), which also differs from the design, but perhaps we should reuse `--studio-jetpack-green-50` for uniformity?

Reply:
BTW; some of the values used for box shadows, spacing font size etc might also exist as variables to make it DRY - I haven't dug deep into that one 😄

Reply:
Nice catch, fixed.